frontend/views/containers/modals/InviteByLink.vue

```diff
@@ -6,26 +6,29 @@ modal-template(ref='modal')
   .c-container
     i18n.title.is-4(tag='h3') Share this link to grant access to your group.
     i18n.has-text-1(tag='p') After the onboarding period has ended, everyone will be asked to vote on whether or not a new member should be added. But for now, enjoy 60 free passes!
-    a.link.has-icon.c-link(:href='link' target='_blank')
-      i.icon-link
-      | {{link}}
-    i18n.has-text-1(tag='p') This invite link expires on the 4th of February.
+    .c-link
+      a.link.has-icon(:href='link' target='_blank' data-test='invitationLink')
+        i.icon-link
+        | {{link}}
+    i18n.has-text-1(tag='p') This invite link expires on the [4th of February].
```

Comment:
`[4th of February]` should be a variable, i.e. `{ expiryDate }`

Reply:
`expiryDate` was not implemented yet... that's why I left a dummy value.

Reply:
Yes, but you can still put the variable there for translators. The value can be fixed later

Reply:
Done.